test/explorer.test.js

```diff
@@ -258,7 +265,7 @@ describe('explorer', function() {
 
     it('can be disabled by configuration', function(done) {
       var app = loopback();
-      app.set('remoting', { cors: { origin: false }});
+      app.set('remoting', { cors: false });
```

Comment:
This change is needed to fix `npm test` when using LB 3.x. Before, the tests were failing because `remoting.cors` was rejected by LB 3.x runtime.